README.md

```diff
@@ -68,3 +68,72 @@ This requires `podman` and `sscg` to be available on the host.
    ```
    make check
    ```
+
+## Running on Kubernetes
+
+The app service can also be deployed on Kubernetes, in particular the
+[cockpit-dev project on the staging instance](https://console-openshift-console.apps.c-rh-c-eph.8p0c.p1.openshiftapps.com/k8s/ns/cockpit-dev/pods). This requires you to be inside the Red Hat VPN. Make sure you select the right project with
+
+    oc project cockpit-dev
+
+There is the [app service image BuildConfig](./webconsoleapp-k8s-buildconfig.yaml) and the [pods and services](./webconsoleapp-k8s.yaml).
+Both get deployed with
+
+    make k8s-deploy
+
+
+1. Validate that you can reach the deployment with:
+
+    curl -u user:password https://test.cloud.redhat.com/api/webconsole/v1/ping
+
+   You should get a "pong" response.
+
+2. Prepare a VM to act as the target machine like above for local podman deployment.
+
+3. The hardest part: Due to a [misconfiguration of 3scale](https://issues.redhat.com/browse/COCKPIT-795?focusedCommentId=20703283&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-20703283), browsers don't ask for basic auth. So you need to set up a local proxy that provides the `Authorization:` header for 3scale:
+
+   - Download Linux binaries from https://mitmproxy.org/, unpack them
+   - Create `add-header.py`:
+   ```python
+   class AddHeader:
+       def __init__(self):
+           self.num = 0
+
+   def request(self, flow):
```

Comment:
This should be `def request(flow):` see https://docs.mitmproxy.org/stable/addons-examples/#http-add-header

Reply:
Hmm, it's what I've always used, with mitmproxy 8.1.1

Reply:
Hah, sorry -- indentation error. fixed!